runtime/errors.go

```diff
@@ -116,6 +115,7 @@ func DefaultHTTPError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w
 	if !ok {
 		s = status.New(codes.Unknown, err.Error())
 	}
+	pb := s.Proto()
```

Comment:
We can replace the code above this like with `status.Convert(err)` (it's equivalent AFAIK).